test/verify/machineslib.py

```diff
@@ -1430,6 +1431,13 @@ def installWithErrorTest(dialog):
                                                          os_name=config.FEDORA_28,
                                                          os_short_id=config.FEDORA_28_SHORTID))
 
+            createDownloadAnOSTest(TestMachines.VmDialog(self, sourceType='downloadOS',
+                                                         is_unattended=True, profile="Workstation",
+                                                         user_password="catsaremybestfr13nds", root_password="dogsaremybestfr13nds",
```

Comment:
You really need to make up your mind! :laughing:

Reply:
More seriously, can there also be a test case which just sets a root (or just a user) password?

Reply:
Sure, will do.

Reply:
Added.